requirements-test.txt

```diff
@@ -1,9 +1,5 @@
 datadog
 freezegun==0.3.11
-msgpack-python<0.5.0
```

Comment:
Wonder why we even had this, it's deprecated.

Reply:
_shrug_